src/bat_utils/output.rs

```diff
@@ -123,7 +123,7 @@ impl OutputType {
                     p
                 };
                 if config.navigate {
-                    process.args(&["--pattern", &navigate::make_navigate_regexp(&config)]);
+                    process.args(&[format!("+r/{}", navigate::make_navigate_regexp(&config))]);
```

Comment:
Since the flag is pretty obscure, consider adding a comment e.g.
> "+" behaves approximately as-if you pressed those keys interactively.
> "r" (repaint the screen) before "/" (search) is helpful to show the diff before "Pattern not found" in case of no match.

Reply:
Done, thanks!